<launch>
    <node pkg="prometheus_detection" type="ellipse_det" name="ellipse_det" output="screen">
        <param name="camera_topic" type="string" value="/P300_Monocular_front/Monocular/image_raw" />
        <param name="camera_info" type="string" value="camera_param_gazebo_monocular.yaml" />
    </node>
</launch>
